src/include/rados.h

```diff
@@ -517,6 +517,7 @@ struct ceph_osd_op {
 			__le64 ver;     /* no longer used */
 			__u8 op;	/* CEPH_OSD_WATCH_OP_* */
 			__u32 gen;      /* registration generation */
+			__u32 timeout; /* connection timeout */
```

Comment:
@ryneli i was wrong the largest struct in the union is
```c++
		struct {
          int64_t offset, length;
          int64_t truncate_size;
          int32_t truncate_seq;
		} __attribute__ ((packed)) extent;
```

so the assert should be
```c++
static_assert(sizeof(ceph_osd_op) == (2+4+(2*8+8+4)+4),
```